src/css/_color-palette.react.css

```diff
@@ -1,17 +1,23 @@
+/* React: Color Palette */
+
 .colorpalette {
     position: absolute;
     z-index: var(--z01-colorpalette);
     right: 18px;
     bottom: 12px;
-    user-select: none;
-    pointer-events: none;
+    display: flex;
+    flex-direction: row;
 }
 
-.colorpalette-element {
+.palette-color {
     display: inline-block;
+    position: relative;
+    padding-left: 2px;
+    padding-right: 2px;
+}
+
+.square {
```

Comment:
For the foreseeable future, CSS class names will be in a global namespace, so let's be better about namespacing CSS class names ourselves to limit the chances of simple class names like `.colors` or `.square` potentially existing amongst multiple components, creating clashes that would be harder to fix modularly in the future.

Let's change `.palette-color` and `.square`. We can stick with `.colorpalette-` as the namespace prefix (unless you have a better suggestion).